src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -292,6 +284,14 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseLayoutMenuItems(Curren
 							ItemType = ItemType.Toggle
 						},
 						new ContextMenuFlyoutItemViewModel()
+						{
+							Text = "SyncStatus".GetLocalizedResource(),
+							IsChecked = itemViewModel.IsSortedBySyncStatus,
+							Command = new RelayCommand(() => itemViewModel.IsSortedBySyncStatus = true),
+							ShowItem = currentInstanceViewModel.IsPageTypeCloudDrive,
+							ItemType = ItemType.Toggle
+						},
+						new ContextMenuFlyoutItemViewModel()
```

Comment:
Why was this changed?

Reply:
This was necessary after reordering the enum (please see below)